google/cloud/sql/connector/InstanceConnectionManager.py

```diff
@@ -379,12 +373,7 @@ def _auth_init(self) -> None:
             ]
         )
 
-        cloudsql = googleapiclient.discovery.build(
-            "sqladmin", "v1beta4", credentials=scoped_credentials
```

Comment:
Similar to other number, putting this version in a constant might be useful, especially if there is/could be multiple builds ever? If not, it is less of a concern.